relayer/pkg/chainlink/txm/txm.go

```diff
@@ -30,6 +29,14 @@ type StarkTXM interface {
 	TxManager
 }
 
+//go:generate mockery --name TxConfig --output ./mocks/ --case=underscore --filename txconfig.go
+
+type TxConfig interface {
+	TxTimeout() time.Duration
+	TxSendFrequency() time.Duration
+	TxMaxBatchSize() int
+}
+
```

Comment:
Consider extracting to `txm/config.go`.

Additionally, there is a [test_helpers.go](https://github.com/smartcontractkit/chainlink-starknet/tree/f4d2e6b4d47188fd5b9fa15fd377fc907910ece3/relayer/pkg/chainlink/txm)/test_helpers.go) file in this package which doesn't really belong here. It should be moved to `ops` or `integration-tests` module, potentially even split between the two.